cmd/stchaind/main.go

```diff
@@ -55,6 +55,8 @@ func main() {
 	rootCmd.AddCommand(genutilcli.ValidateGenesisCmd(ctx, cdc, app.ModuleBasics))
 	rootCmd.AddCommand(AddGenesisAccountCmd(ctx, cdc, app.DefaultNodeHome, app.DefaultCLIHome))
 	rootCmd.AddCommand(AddFaucetCmd(ctx, cdc, app.DefaultNodeHome, app.DefaultCLIHome))
+	rootCmd.AddCommand(AddLoadTestCmd(ctx, cdc, app.DefaultNodeHome, app.DefaultCLIHome))
+	rootCmd.AddCommand(AddRandomLoadTestCmd(ctx, cdc, app.DefaultNodeHome, app.DefaultCLIHome))
```

Comment:
you should make it 1 cmd with a flag to make it random or not

Reply:
Re-organized the command struc. Cuz load test using random/fixed senders have different flag sets (--init-distribution and others), I just make random/fixed as sub-commands of "stchaind load", think this is also intuitive to use.

Usage:
  stchaind load [command]

Available Commands:
  fixed       Run a load test with fixed senders and receivers
  random      Run a load test with random senders and receivers